src/XMakeTasks/UnitTests/GenerateResource_Tests.cs

```diff
@@ -2071,6 +2071,7 @@ public void SourceItemMissing()
 #else
         [Fact(Skip = "https://github.com/Microsoft/msbuild/issues/297")]
 #endif
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
VIa this conversation maybe we can also revisit #297 for some more investigation / resolution.